vendor/legacy-each-view-item-controller-support/index.js

```diff
@@ -0,0 +1,26 @@
+if (Ember.ENV._ENABLE_LEGACY_VIEW_SUPPORT) {
```

Comment:
I left a runtime check in case you toggle the flag off without restarting ember serve.